src/classes/BDI_DataImportService.cls

```diff
@@ -77,13 +77,13 @@ global with sharing class BDI_DataImportService {
     * @param batchId id of batch to load
     * @return DataImportBatch__c
     */
-    private static DataImportBatch__c loadBatch(ID batchId) {
+    private static DataImportBatch__c loadBatch(Id batchId) {
 
-        List<DataImportBatch__c> listDIBatch = BDI_DataImportBatch_SEL.selectByIds(new Set<Id>{batchId});
+        List<DataImportBatch__c> listDIBatch = BDI_DataImportBatch_SEL.selectByIds(new Set<Id>{ batchId });
         if (listDIBatch.size() == 1) {
             return listDIBatch[0];
         } else {
-            throw(new BDIException(String.format(Label.bdiInvalidBatchId, new String[]{batchId})));
+            throw(new BDIException(String.format(Label.bdiInvalidBatchId, new String[]{ batchId })));
```

Comment:
This and other labels should be `System.Label. bdiInvalidBatchId`

Reply:
Also, there's an inconsistency in how the `String.format()` second parm is instantiated throughout this class. Sometimes it's `String[]` and sometimes it's `new List<String>`. I'm more of a fan of the latter, but in the end consistency is better.